content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,702 @@
+      - :guilabel:`Average`: this row provides the average of all other rows in the column.
+
+      The cohort view can also be downloaded as an Excel file, by clicking the :guilabel:`Download`
+      icon in the top-left of the page.
+
+   .. tab:: List View
+
+      The list view displays a single list of all leads/opportunities matching the search criteria.
+      Clicking a lead/opportunity opens the record for closer review.
+
+      This view option is useful for reviewing many records at once.
+
+      .. image:: win_loss/list-view.png
+         :align: center
+         :alt: The List View displays a single list of all records matching the search criteria.
+
+      Additional details for each record such as :guilabel:`Country`, :guilabel:`Medium`, and more
+      can be added to the list, by clicking the :guilabel:`Filters` icon in the top-right of the
+      list.
+
+      .. image:: win_loss/list-view-columns.png
+         :align: center
+         :alt: More details can be added to the list by clicking the Filters icon in the top-right.
+
+      Clicking the :guilabel:`⚙️ (gear)` icon opens the Actions drop-down menu, with options for the
+      following:
+
+      - :guilabel:`Import records`: opens a page for uploading a spreadsheet of data, as well as a
+        template spreadsheet to easily format that data.
+      - :guilabel:`Export All`: downloads the list as an xlsx file for Excel.
+      - :guilabel:`Knowledge`: inserts a view of, or link to, the list in an article in the
+        *Knowledge* app.
+      - :guilabel:`Dashboard`: adds the list to *My Dashboard* in the *Dashboards* app.
+      - :guilabel:`Spreadsheet`: links to, or inserts, the list in a spreadsheet in the *Documents*
+        app.
+
+      .. note::
+         On the list view, clicking :guilabel:`New` closes the list, and opens the *New Quotation*
+         page. Clicking :guilabel:`Generate Leads` opens a pop-up window for lead generation.
+         Neither feature is intended to manipulate the list view.
+
+.. _win_loss/reports:
+
+Create reports
+==============
+
+After understanding how to :ref:`navigate the pipeline analysis page <win_loss/pipeline>`, the
+:guilabel:`Pipeline Analysis` page can be used to create and share different reports. Between the
+pre-made options and custom filter/groupings, almost any combination is possible.
+
+Once created, reports can be :ref:`saved to favorites, shared with other users, and/or added to
+dashboards and spreadsheets <win_loss/save_reports>`.
+
+A few common reports that can be created using the :guilabel:`Pipeline Analysis` page are detailed
+below.
+
+.. _win_loss/win_loss:
+
+Win/Loss reports
+----------------
+
+Win/Loss is a metric of the leads/opportunities marked "Won" or "Lost" over a period of time. By
+calculating "opportunities won" over "opportunities lost," teams can identify what methods and/or
+individuals are effectively converting leads into sales.
```

Comment:
the punctuation in this section is unnecessary and distracting...
here's how I would simplify the punctuation and expand on the lesson that's trying to be communicated here.

```suggestion
Win/Loss is a calculation of active or previously active leads in a pipeline that were either marked
as **Won** or **Lost** over a specific period of time. By calculating opportunities won over
opportunities lost, teams can identify clarify key performance indicators (KPIs) that are converting
leads into sales, such as specific teams or team members, certain marketing mediums or campaigns,
and so on.
```